### PR TITLE
[ECF-1-307] Admins edit admins

### PR DESCRIPTION
**Pull request by amazimbe**
### Context
Allow admins to edit admins

### CODE REVIEW

File: app/views/layouts/admin/base.html.erb [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <%= component.nav_item(path: "/admin/administrators") do %>
       Administrators
     <% end %>
+    <%= component.nav_item(path: "/admin/induction-coordinators") do %>
+      Induction Coordinators
+    <% end %>
```
Comment (kicferk1) [inline app/views/layouts/admin/base.html.erb:11]: Good spot

**Review by kicferk1 — APPROVED** [review]

File: app/views/admin/administrators/administrators/index.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
       <tbody class="govuk-table__body">
         <% @administrators.each do |administrator| %>
           <tr class="govuk-table__row">
-            <td class="govuk-table__cell"><%= administrator.full_name %></td>
+            <td class="govuk-table__cell"><%= govuk_link_to administrator.full_name, edit_admin_administrator_path(administrator) %></td>
```
Comment (ethanmills) [inline app/views/admin/administrators/administrators/index.html.erb:15]: Are we not using the helper here? Why only in some places?

**Review by ethanmills — APPROVED** [review]
Looks good, just one question

File: app/views/admin/administrators/administrators/index.html.erb (hunk 2 above)
Comment (amazimbe) [reply app/views/admin/administrators/administrators/index.html.erb:15]: I don't think it's strictly needed. My main motivation for the helper was on the suppliers page where we have a list that mixes up delivery partner users and lead provider users and I needed to send each user to the right endpoint. The admins and induction coordinators don't get combined that why I haven't used it here. @ethanmills

Comment (ethanmills) [reply app/views/admin/administrators/administrators/index.html.erb:15]: Fair enough. You do use it for induction coordinators though, right? https://github.com/DFE-Digital/early-careers-framework/pull/122/files#diff-9e81e35702dad35ed3b19ae78f92ad76e63185d92a3158b0818ba73110b36927R14

Comment (ethanmills) [reply app/views/admin/administrators/administrators/index.html.erb:15]: Ok 🙂 Happy for this to go in as-is

File: app/views/layouts/admin/base.html.erb (hunk 1 above)
Comment (seb-siem-softwire) [reply app/views/layouts/admin/base.html.erb:11]: Nice

**Pull request merged**